backend/api/auth.py

```diff
@@ -41,3 +42,13 @@ def verify_user(session=Depends(get_request_session)):
         )
 
     return maybe_user
+
+
+def verify_admin(current_user: Annotated[dict, Depends(verify_user)]):
+    if not current_user["is_admin"]:
```

Comment:
It might be nice as we're adding some of this RBAC functionality to include test cases that capture enforcement (e.g. that a non-admin / manager trying to call an endpoint gets the expected `403`). Maybe these can go in a dedicated test file, but it may help us with peace of mind that our permissions are working as expected / desired :joy: .